backend/molgenis-emx2-beacon-v2/src/main/java/org/molgenis/emx2/beaconv2/Templates.java

```diff
@@ -0,0 +1,58 @@
+package org.molgenis.emx2.beaconv2;
+
+import graphql.ExecutionInput;
+import graphql.GraphQL;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+import org.molgenis.emx2.Database;
+import org.molgenis.emx2.Schema;
+import org.molgenis.emx2.graphql.GraphqlApiFactory;
+
+public class Templates {
+
+  public static void addTemplatesToDb(Database database) {
+    database.becomeAdmin();
+    Schema schema = database.getSchema("_SYSTEM_");
+    GraphQL graphQL = new GraphqlApiFactory().createGraphqlForSchema(schema);
+
+    for (EntryType entryType : EntryType.values()) {
+      String query =
+          "mutation insert($endpoint:String, $schema:String, $template:String) {"
```

Comment:
I think you should here simply use Database.getSchema().getTable().insert(Row...)